### PR TITLE
Adds in `gridTemplateAreas` and `gridArea`

### PR DESCRIPTION
**Pull request by drew887**
For #156; I took another look and decided I should try and implement it myself if I couldn't find it :slightly_smiling_face:; happy to implement any required changes if need be. 

Also don't quite know how I really feel about the `areas` name for the not `none` constructor but I don't know a better name off hand.

### CODE REVIEW

**Comment by giraud** [comment]
Thanks for your PR

File: src/Css.rei [hunk 1]
```diff
@@ -667,6 +667,8 @@ let flex3:
 let flexBasis: [ Types.FlexBasis.t | Types.Length.t] => rule;
 let gridAutoColumns: [ Types.Length.t | `auto] => rule;
 let gridAutoRows: [ Types.Length.t | `auto] => rule;
+let gridArea: string => rule;
```
Comment (giraud) [inline src/Css.rei]: For grid area, given its definition in https://developer.mozilla.org/fr/docs/Web/CSS/grid-area, I think something like this is more accurate:
```
module GridArea: {
  type t = [ `auto | `ident(string) | `num(int) | `numIdent(int, string) | `span([ | `num(int) | `ident(string) ])];
}

gridArea:  [ GridArea.t | Cascading.t ] => rule
gridArea2: (GridArea.t, GridArea.t) => rule;
gridArea3: (GridArea.t, GridArea.t, GridArea.t) => rule;
gridArea4: (GridArea.t, GridArea.t, GridArea.t, GridArea.t) => rule;
```

File: src/Css.rei [hunk 2]
```diff
@@ -667,6 +667,8 @@ let flex3:
...
+let gridTemplateAreas: Types.GridTemplateAreas.t => rule;
```
Comment (giraud) [inline src/Css.rei]: you are missing the Cascading type

File: src/Css.rei (hunk 1 above)
Comment (drew887) [reply src/Css.rei]: I honestly didn't even know the `/` syntax for grid-area was a thing! :astonished: 

Those make sense though, thank you will add!

**Comment by giraud** [comment]
Perfect

**Pull request merged**